mix.exs

```diff
@@ -58,4 +86,53 @@ defmodule CouchDBTest.Mixfile do
   def get_test_paths(_) do
     []
   end
+
+  defp get_deps_paths() do
```

Comment:
I have mixed fillings about this. We could just hardcode the dependencies we want to include in report. The rationale against hardcoding:

* we want to have a report for any new application by default (because I am sure we forget to update the list)